### DISCUSSION TITLE
Orchard Core and user management app

### DISCUSSION THREAD

**Discussion by moco32** [OrchardCMS/OrchardCore#10895] [Q&A] [answered]
Is it possible to connect an orchard core app to an already existing user management app? I'd like to login in the orchard core app but verify the user and check it's role in an already existing user management and according to what the user management returns, I want to show the corresponding content for that user in the orchard core app. Anyone knows if anything like this has been done?

**Comment by hishamco** [comment]
You can implement your own stores, ASP.NET Core Identity is much flexible and extensible

**Reply by moco32** [reply]
@hishamco can you point me in a direction of where to find tutorials for something like this?

**Reply by hishamco** [reply]
Please check the `UserStore` implementation

**Comment by ns8482e** [comment, marked as answer]
@moco32  if your user management app suports  Open ID/OAuth -then  you can connect with orchard 

You may need to create a module for your external app, However it will be similar implementation like other external identity providers (Google, Microsoft, Github) - provided out of box.

Here's link for Azure AD doc for your reference - https://docs.orchardcore.net/en/latest/docs/guides/azuread-integration/

**Reply by hishamco** [reply]
@moco32 Niraj points to a good option that I forgot to mention

**Reply by moco32** [reply]
Thank you @ns8482e
